apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/lib/validation.ts

```diff
@@ -266,3 +273,218 @@ export const isSurveyLogicCyclic = (questions: TSurveyQuestions) => {
 
   return false;
 };
+
+export const validateSurvey = (
+  survey: TSurvey,
```

Comment:
Can we rename this function to `isSurveyValid` or something else? As it returns boolean, renaming would make more sense.